cmd/generate.go

```diff
@@ -26,14 +28,19 @@ This Program generates step for the provided plugins with configurations
 
 func init() {
 	cobra.OnInitialize(initConfig)
-	generateCmd.PersistentFlags().StringVar(&ConfigFilePath, "config", "conf.yaml", "Mention the config file path")
+	generateCmd.PersistentFlags().StringVar(&ConfigFilePath, "config", "", fmt.Sprintf("Mention the config file path (default \"%s\")", defaultConfigFilePath))
 }
 
 func initConfig() {
 	if ConfigFilePath != "" {
 		log.Debug("Config Path:", ConfigFilePath)
 		if err := config.LoadConfig(ConfigFilePath); err != nil {
-			log.Warn("Error while loading the configuration file. Loading the defaults")
+			log.Fatal("Error while loading the configuration file. Exiting the program.")
+		}
+	} else {
```

Comment:
for code readability, we can avoid a number of `else` in general by breaking early - eg:
```
	if ConfigFilePath != "" {
		log.Debug("using specified config file: ", ConfigFilePath)
		if err := config.LoadConfig(ConfigFilePath); err != nil {
			log.Fatal("error while loading the configuration file (will exit): ", err)
		}
		return
	}
	log.Debug("using default config file: ", defaultConfigFilePath)
	if err := config.LoadConfig(defaultConfigFilePath); err != nil {
		log.Debug("error while loading the configuration file (will load the defaults): ", err)
	}
```

Reply:
done